src/tasks/filetypes/eigensols.rs

```diff
@@ -16,7 +16,7 @@ use crate::traits::{Load, AsPath, save::Json};
 
 // Conversion factor phonopy uses to scale the eigenvalues to THz angular momentum.
 //    = sqrt(eV/amu)/angstrom/(2*pi)/THz
-const SQRT_EIGENVALUE_TO_THZ: f64 = 15.6333043006705;
+const SQRT_EIGENVALUE_TO_THZ: f64 = 15.6333043006705 * 0.208;
```

Comment:
Yikes!  We'll definitely need to figure out a better way to adjust this.

(also, you should document where the 0.208 comes from)